docs/intro/getting-started-with-sphinx.rst

```diff
@@ -120,7 +116,9 @@ Here are some external resources to help you learn more about Sphinx.
 
 * `Sphinx documentation`_
 * :doc:`RestructuredText primer <sphinx:usage/restructuredtext/basics>`
+* `Getting Started with MyST`_
```

Comment:
We could move this link to the `Using Markdown with Sphinx` section so is easy to find. Something like `You can use Markdown using MyST_`

Reply:
Sure 👍🏾